store/src/main/java/org/apache/rocketmq/store/MessageStore.java

```diff
@@ -118,7 +118,7 @@ GetMessageResult getMessage(final String group, final String topic, final int qu
      * @param timestamp Timestamp to look up.
```

Comment:
Missing the JavaDoc of new param `isGetTimeLast` here. BTW, `isGetTimeLast` is confused~

Reply:
Yes,and the JavaDoc if param isGetTimeLast has been added